src/components/MerchantsPage/styles.module.scss

```diff
@@ -152,6 +152,39 @@
   }
 }
 
+.bannerContentContainer {
+  // grid relative to parent container
+  grid-row: 1;
+  grid-column: 1;
+  z-index: 1;
```

Comment:
Why do we need to change the z-index?

Reply:
Not setting a z-index causes the content to get hidden behind the banner 😬

Reply:
This is a bit of code smell 😬